### DISCUSSION TITLE
chain commands in click-left with `&&`

### DISCUSSION THREAD

**Discussion by mil-ad** [polybar/polybar#3084] [Q&A]
Is there a way to chain commands with `&&` or `||` when passing a shell command to things like `click-left` (in Text module for example)?

I can pipe things just fine:

```
click-left = program_1 | program_2
```

but I'm trying to do something like

```
click-left = program_1 | program_2 && program_3
```

I tried grouping with `{ }` which doesn't create a new subshell or passing everything in a `bash -c " ... "` but haven't managed to make it work

**Comment by patrick96** [comment]
Polybar executes everything in `click-left` as `/bin/sh -c "..."` (or `$POLYBAR_SHELL` if it is set). So what you are doing should work, assuming `program_2` has a zero exit code.

Try playing around in the terminal to see if something may be going wrong.
